doc/mkdocs/docs/css/custom.css

```diff
@@ -0,0 +1,4 @@
+/* disable ligatures in code and preformatted blocks */
+code, pre {
+    font-variant-ligatures: none;
+}
```

Comment:
The chosen mkdocs font supports ligatures and messes with `<=>`, etc.

Reply:
You mean like

<img width="426" alt="image" src="https://user-images.githubusercontent.com/159488/164782229-1de6910b-7ac9-4ce0-8633-0830a1e9bd81.png">

?

Reply:
Exactly.